les/odr_requests.go

```diff
@@ -401,32 +348,6 @@ func (r *ChtRequest) Validate(db ethdb.Database, msg *Msg) error {
 	log.Debug("Validating CHT", "cht", r.ChtNum, "block", r.BlockNum)
 
 	switch msg.MsgType {
-	case MsgHeaderProofs: // LES/1 backwards compatibility
```

Comment:
Can we also remove the switch statement here and add a if check if msg.MsgType != MsgProofsV2 then ...
Just for the code style consistency.

Reply:
done